src/includes/class-alma-wc-refund-helper.php

```diff
@@ -184,7 +184,85 @@ private function has_transaction_id( $order ) {
 	public function format_refund_comment( $comment ) {
 		return self::PREFIX_REFUND_COMMENT . $comment;
 	}
-}
 
+	/**
+	 * Adds an alma_refund_notices in post_meta
+	 *
+	 * @param WC_Order $order The order where to add a note & notice.
+	 * @param string   $notice_type Notice type.
+	 * @param string   $message The message.
+	 *
+	 * @return void
+	 */
+	protected function add_notice( WC_Order $order, $notice_type, $message ) {
+		$refund_notices   = get_post_meta( $order->get_id(), self::REFUND_NOTICE_META_KEY, false );
+		$refund_notices[] = array(
+			'notice_type' => $notice_type,
+			'message'     => $message,
+		);
+		update_post_meta( $order->get_id(), self::REFUND_NOTICE_META_KEY, $refund_notices );
+	}
 
+	/**
+	 * Adds a refund error note to an order + a notice
+	 *
+	 * @param WC_Order $order The order where to add a note & notice.
+	 * @param string   $message The message.
+	 *
+	 * @see add_order_note()
+	 * @return void
+	 */
+	public function add_error_note( WC_Order $order, $message ) {
+		$this->add_error_note( $order, $message );
+	}
 
+	/**
+	 * Adds a refund success note to an order + a notice
+	 *
+	 * @param WC_Order $order The order where to add a note & notice.
+	 * @param string   $message The message.
+	 *
+	 * @see add_order_note()
+	 * @return void
+	 */
+	public function add_success_note( WC_Order $order, $message ) {
+		$this->add_success_note( $order, $message );
+	}
+
+	/**
+	 * Print refund notices previously stored as HTML format
+	 *
+	 * @param WC_Order $order The order linked to the refund notices.
+	 *
+	 * @return void
+	 */
+	public function print_notices( WC_Order $order ) {
+		$refund_notices = get_post_meta( $order->get_id(), 'alma_refund_notices', true );
```

Comment:
Should use `self::REFUND_NOTICE_META_KEY` instead of `alma_refund_notices` here.
(I corrected this in a commit)